rapid_response_xblock/block.py

```diff
@@ -215,10 +215,9 @@ def get_lms_template_context(self):
             course_key=self.course_key,
             open=True
         ).exists()
-        return LmsTemplateContext(
-            is_open=is_open,
-            is_staff=self.is_staff()
-        )._asdict()
+        return {
```

Comment:
`is_staff` is removed because we already verify that `is_staff` is true in the decorator for the views

Reply:
Yup, makes sense. Might be best just to get rid of `get_lms_template_context` at this point and just leave a helper method to run that query at line 213 (`has_open_run` or something)